composer.json

```diff
@@ -54,7 +54,7 @@
         "symfony/mailer": "^7.0",
         "symfony/mime": "^7.0",
         "symfony/translation": "^7.0",
-        "symfony/validator": "^7.1",
+        "symfony/validator": "^7.2",
```

Comment:
Needed because the visibility of [PasswordStrengthValidator::estimateStrength()](https://github.com/symfony/symfony/blob/7.2/src/Symfony/Component/Validator/Constraints/PasswordStrengthValidator.php#L60) changed from private to public in 7.2